### PR TITLE
New Exercise: Rarity of Strings

### PR DESCRIPTION
**Pull request by Ennowulff**
I tried my best to create my first exercism...

### CODE REVIEW

File: exercises/concept/rarity_of_string/.meta/config.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "blurb": "Calculate the rarity of a string",
+  "authors": ["ennowulff"],
+  "contributors": [
+  ],
+  "files": {
+    "solution": ["zcl_rarity_of_String.clas.abap"],
```
Comment (mbtools) [inline exercises/concept/rarity_of_string/.meta/config.json:7]: 2x zcl_rarity_of_String > zcl_rarity_of_string

File: exercises/concept/rarity_of_string/.meta/zcl_rarity_of_String.abap [hunk 2]
```diff
@@ -0,0 +1,29 @@
+CLASS zcl_rarity_of_string DEFINITION
```
Comment (mbtools) [inline exercises/concept/rarity_of_string/.meta/zcl_rarity_of_String.abap:1]: rename file zcl_rarity_of_String > zcl_rarity_of_string

**Review by mbtools — CHANGES_REQUESTED** [review]
Already pretty good. A few things...

- The exercise should go into the `exercises/practice` folder (not `concepts` since we don't have those yet)
- The file in `.meta` needs to be renamed. There's currently no better way than manually adding the `.meta`
- There are some changes to unrelated exercises in the `practice` folder. Those were caused by pushing from ABAP I assume. Please revert those changes to avoid a warning for all users who have already worked on the exercises. 

It looks like the exercise is new to the Exercism platform (it's not from https://github.com/exercism/problem-specifications/). We will need to request a new icon.

**Comment by Ennowulff** [comment]
> Already pretty good. A few things...

> * The exercise should go into the `exercises/practice` folder (not `concepts` since we don't have those yet)
Okay. I thought "concepts" is for new exercises to be reviewed

> * The file in `.meta` needs to be renamed. There's currently no better way than manually adding the `.meta`
Done.

> * There are some changes to unrelated exercises in the `practice` folder. Those were caused by pushing from ABAP I assume. Please revert those changes to avoid a warning for all users who have already worked on the exercises.
I don't understand what you mean 😢

> It looks like the exercise is new to the Exercism platform (it's not from https://github.com/exercism/problem-specifications/). We will need to request a new icon.
Of course it's new. That's the reason for the fork... 😀 Right, a new icon would be fine. Is there a How-To for new exercises?

**Comment by mbtools** [comment]
please commit the changes to your branch :-)

With "new exercise" I meant that it does not exist in any programming language on Exercism. Example for requesting new icon:
https://github.com/exercism/website-icons/issues/126

**Comment by mbtools** [comment]
bin im urlaub und melde mich naechste woche

**Pull request closed without merging**